.github/workflows/budibase_ci.yml

```diff
@@ -25,6 +25,13 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
+      - name: Maximize build space
```

Comment:
This is mentioned in the PR - what is it for?

Reply:
Unrelated to the main body of work - hoping this helps with test stability - I merged it to develop and tests were flaky on master, so added it to this PR to see if it helps here.

Reply:
PR for `develop`: https://github.com/Budibase/budibase/pull/11760

Mentioned it in engineering, won't cause any issues for releases etc - if its a problem on `master` i'll roll it back, but trying to get it more consistent, had to run a master PR 7 times yesterday to get it to pass!

Reply:
Cool - hopefully it does help! 😄